Makefile

```diff
@@ -92,6 +92,7 @@ unused-libs:
 .PHONY: core
 core:
 	$(MAKE) minimal-build
+	strip semgrep-core
```

Comment:
this will have no effect; semgrep-core is not in the current directory.
Did you try 'make core'?

Reply:
I haven't tried it locally as my current build env is broken (with missing libraries that should not be missing), so I was relying on the workflows to test, as they upload the artifact 😄 

But indeed I read the `ln -s` line backwards! Updated (and I'll try to fix my local env)

Reply:
Now I managed to run `make core` and confirm strip is working however it feels like there should be a better option in this build system (that I'm not familiar) to hook into...

Reply:
@aryx sorry about all the mess, I still don't understand how `ln` seems to run with a different cwd than the command after (since `semgrep-core` does not exist there).

I've updated to use the full target path and also tested it in a PR directly in my fork ✅